### ISSUE TITLE
DITA to various flavors of CommonMark/Markdown

### ISSUE THREAD

**Issue by wbrisett** [jelovirt/org.lwdita#24]
In DITA tables we have a lot of flexibility. For example, we can do this:

```xml
<entry colname="col3">
	<p>Generic text here.</p>
	<p>More generic text here. </p>
	<dl compact="yes">
		<dlentry>
			<dt>0</dt>
			<dd><p>There's a lot going on here.</p></dd>
		</dlentry>
		<dlentry>
			<dt>1</dt>
			<dd><p>More content is placed here.</p></dd>
		</dlentry>
	</dl>
	<p>More explanation goes here.</p>
</entry>
```

To be fair, this falls within the CommonMark/Markdown limitations. But what I would hope to get out is a table with some representation of that data. However, that's not what happens. I end up with a table entry that has the first paragraph item in it, then the table pretty much breaks at that point and the remaining content is placed in the file, but not in the table.

## Possible Solution (enhancement)

Multiple lines are extremely hard to deal with in Markdown tables. In fact, the only solution I know of is to make tables HTML tables instead of MD text.

**Comment by jelovirt** [comment]
Using HTML5 tables for output is the only real solution here, but then all the content will need to be HTML5.

**Comment by 1ceb3rg** [comment]
Is it possible to just output the table as HTML5? Vuepress and many MD tools support inline HTML.

**Comment by jelovirt** [comment]
Implemented in #200
